pages/episodes/[slug].vue

```diff
@@ -15,32 +15,32 @@ useHead({
   bodyAttrs: {
     class: 'has-head-color'
   }
-})
-const dataLoaded = ref(false)
-const episode = ref([])
+} )
+const dataLoaded = ref( false )
```

Comment:
can't you just determine if data was loaded by looking at the length of episode? (I think this is also done above)

Reply:
we've been using this pattern throughout the site so I'd keep it for consistency. we plan to use the fetch hook when it's available anyway, this this will go away eventually

Reply:
It sounds like introducing it here is just more work anyways, having 2 variables like this permits these variables to become out-of-sync (even though it may seem impossible based on the code now).

Reply:
we're switching to fetch hook anyway, so this will go away soon